packages/components/src/button/test/index.tsx

```diff
@@ -138,7 +134,7 @@ describe( 'Button', () => {
 
 			expect( tooltip ).toBeVisible();
 
-			await user.tab();
+			await press.Tab();
```

Comment:
Noting that these have historically been flaky and we've prepended a `await sleep();` before to resolve that flakiness. Not perfect, of course, and ideally, it would be nice to find a cleaner way that will allow us to clean the existing instances up (see https://github.com/WordPress/gutenberg/issues/59071)

Reply:
good note, something to look into - might be something worth solving at the `@ariakit/test` level

Reply:
just remembered the context behind this - Diego did mention this should be solved now so probably worth removing those sleeps to see if things work properly now 👍

Reply:
will do in a follow-up, but fyi removing all `await sleep()` calls works!

![image](https://github.com/user-attachments/assets/49b3d562-0985-4647-a2f0-196529b78183)

Reply:
Sounds good! Assigning you #59071 so you can close that, too, with your follow-up.